.gitignore

```diff
@@ -89,4 +89,5 @@ lint/tmp/
 
 # Android Profiling
 *.hprof
-/app/src/main/java/com/example/spacex_demo_app/api/text.txt
+/app/src/main/java/com/example/spacex_demo_app/SimpleRx.kt
```

Comment:
These should not be in .gitignore file. If you don't want to add these files in remote repo just don't add in a commit